dashboard/src/components/AppView/AppControls.test.tsx

```diff
@@ -71,7 +71,7 @@ it("calls delete function with additional purge", () => {
   const app = new hapi.release.Release({ name, namespace });
   const deleteApp = jest.fn(() => false); // Return "false" to avoid redirect when mounting
   // mount() is necessary to render the Modal
-  const wrapper = mount(<AppControls app={app} deleteApp={deleteApp} />);
+  const wrapper = mount(<AppControls app={app} deleteApp={deleteApp} ariaHideApp={false} />);
```

Comment:
I believe we should fix the app element issue in the test instead of allowing developers to disable accessibility properties in the component. Also, we're adding some logic around the final component implementation that only will be used in tests.

The `appElement` is the root node where all the content belongs to. A possible solution is to create a `div` directly in all the tests that uses `ReactModal`:

```
ReactModal.setAppElement(document.createElement('div'));
```

You've another workarounds in https://github.com/reactjs/react-modal/issues/632

Reply:
that worked! thanks @Angelmmiguel